docs/mine/lotus/miner-lifecycle.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Lotus Miner: lifecycle'
+description: 'How to safely perform maintenance on a Lotus miner.'
+breadcrumb: Lifecycle'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+These operations are normally related to maintenances and upgrades. Given that miners are expected to submit proofs to the chain in a continuous fashion while running long and expensive operations, it is important that operators become familiar with how to manage some of the events in the miner lifecycle so that they can be performed with the maximum guarantees.
+
+[[TOC]]
+
+## Safely restarting the miner daemon
+
+Before shutting down your miner daemon, make sure that there are no pending operations that could get your miner _slashed_.
```

Comment:
It would be nice to have some expectation-setting like "A simple restart usually takes about X minutes" and any other caveats around time.

Reply:
A restart will take as long as it takes for your machine to verify the proof params

Reply:
As such it would be a good suggestion that miners put the params onto nvme drives so that they can be quickly read.

Reply:
added this suggestion to the setup instructions.